flake-modules/dev/launch-test.sh

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env bash
+
+: "${NIXVIM_NIX_COMMAND:=nix}"
+if [[ -z $NIXVIM_SYSTEM ]]; then
+	NIXVIM_SYSTEM=$(nix eval --raw --impure --expr "builtins.currentSystem")
+fi
+
+help() {
+	cat <<EOF
+Usage: tests [-i/--interactive] [-h/--help] [-t/test <test name>]
+
+Options:
+	-i/--interactive: Pick interactively the tests
+	-h/--help: Display this help message and exit
+	-t/--test <test name>: Only launch the specified test (conflicts with -i)
+EOF
+}
+
+if ! OPTS=$(getopt -o "hit:" -l "help,interactive,test:" -- "$@"); then
+	echo "Invalid options" >&2
+	help
+	exit 1
+fi
+
+eval set -- "$OPTS"
+
+test_name=
+interactive=false
+
+while true; do
+	case "$1" in
+	-h | --help)
+		help
+		exit 0
+		;;
+	-i | --interactive)
+		interactive=true
+		shift
+		;;
+	-t | --test)
+		test_name=$2
```

Comment:
Can we add a TODO: allow specifying `--test` multiple times?

Or could we just use non-option arguments as test names?

 I.e. using the `*)` case, but after filtering out unrecognised `-*)` options.

Reply:
This could be (TODO) compatible with `-i` by making the provided test-names on by default? Or by enabling them in addition to the interactively enabled tests?

Reply:
(feel free to ignore me until this is undrafted)